style.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,900;1,100;1,200;1,300;1,400;1,800&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url('./imgs/bg.jpg');
+  min-height: 100vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.count-down-container {
+  display: flex;
+}
+
+ul {
+  list-style: none;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+h1 {
+  color: #f03b6d;
+  font-size: 56px;
+  margin-top: -70px;
+}
+
+li {
+  font-size: 35px;
+  margin: 20px 30px;
+  font-weight: 500;
+}
+
+.count {
+  font-weight: 800;
+  font-size: 50px;
+}
+
+.quote {
+  font-size: 40px;
+  font-weight: 900;
+  color: #1729f5;
+  width: 80%;
+  text-align: center;
+  border: 2px solid #1729f5;
+  border-radius: 16px;
+  background-color: #fff;
+  opacity: 0.9;
+}
```

Comment:
I think you should give padding to this container because if you do, it will make the content more readable and attractive.